Frends.Files.Move/Frends.Files.Move.Tests/WindowsServer/docker-compose.yml

```
@@ -0,0 +1,60 @@
+# docker-compose-windows.yml
+version: '3.8'
+services:
+  windows-fileserver:
+    image: mcr.microsoft.com/windows/servercore:ltsc2022
+    container_name: windows_fileserver
+    hostname: fileserver
+    volumes:
+      - fileserver_data:C:\SharedData
+    networks:
+      - test-net
+    command: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)